source/ui/REIXS/REIXSSampleManagementPre2013Widget.cpp

```diff
@@ -0,0 +1,86 @@
+#include "REIXSSampleManagementPre2013Widget.h"
+
+#include <QGridLayout>
+#include <QUrl>
+#include <QGroupBox>
+#include <QSpacerItem>
+
+#include "ui/AMTopFrame.h"
+
+#include "ui/AMBeamlineCameraWidget.h"
+
+REIXSSampleManagementPre2013Widget::REIXSSampleManagementPre2013Widget(QWidget *manipulatorWidget, const QUrl& sampleCamera1Url, const QUrl& sampleCamera2Url, AMSamplePlatePre2013* samplePlate, AMSampleManipulator* manipulator, QWidget *parent) :
+	QWidget(parent)
+{
+#ifdef AM_MOBILITY_VIDEO_ENABLED
+
+
+
+
+	cameraWidget1_ = new AMBeamlineCameraWidget(this, false);
+	cameraWidget1_->playSource(sampleCamera1Url);
+	cameraWidget1_->setCrosshairCenterPosition(QPointF(0.75, 0.6));
+	cameraWidget1_->setCrosshairPosition(QPointF(0.75, 0.6));
+	cameraWidget1_->setCrosshairVisible(true);
+	cameraWidget1_->setCrosshairLocked(true);
+	cameraWidget1_->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
+
+
+	cameraWidget2_ = new AMBeamlineCameraWidget(this, false);
+	cameraWidget2_->playSource(sampleCamera2Url);
+	cameraWidget2_->setCrosshairCenterPosition(QPointF(0.5, 0.7));
+	cameraWidget2_->setCrosshairPosition(QPointF(0.5, 0.7));
+	cameraWidget2_->setCrosshairVisible(true);
+	cameraWidget2_->setCrosshairLocked(true);
+	cameraWidget2_->setSizePolicy(QSizePolicy::MinimumExpanding, QSizePolicy::MinimumExpanding);
+
+#else
+	Q_UNUSED(sampleCameraUrl1);
+	Q_UNUSED(sampleCameraUrl2);
+#endif
+
+	topFrame_ = new AMTopFrame("Sample Management & Positioning", QIcon(":/system-software-update.png"));
```

Comment:
I believe this one would be easier to use the `AMMainWindow::buildMainWindowPane()` method.